nwbwidgets/icephys.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from ipywidgets import widgets
+import itkwidgets
+import itk
+from scipy.signal import stft
+from pynwb.ecephys import LFP
+from .base import fig2widget
+
+
+def show_patchclamp_series(node, **kwargs):
+    ntabs = 2
+    # Use Rendering... as a placeholder
```

Comment:
Please do not implement the tabs with in the function, but instead use an OrderedDict on the neurodata_vis_spec object, like this:

https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/blob/b3841d92a5deb73ac91f65db27f346f7d27da2e7/nwbwidgets/view.py#L10-L12

This is preferred because it separates the lazy tabs logic from the specific visualizations and lets you add new visualizations in a more modular way